### PR TITLE
feat: added wishlist

### PR DESCRIPTION
**Pull request by nakulsharma15**
Created wishlist page

index-1
index-2

### CODE REVIEW

**Review by Karan-Bharti — APPROVED** [review]
Seems good to me, you may merge

File: pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="../style.css">
+
+    <link rel="stylesheet" href="../stylesheets/wishlist.css">
+
```
Comment (shuntzhere) [inline pages/wishlist.html:12]: Can this stylesheet be moved to the main file?

**Review by shuntzhere — APPROVED** [review]
Looks good, classes are well defined. Just remove the extra lines and indent the code.

File: pages/wishlist.html (hunk 1 above)
Comment (nakulsharma15) [reply pages/wishlist.html:12]: I'll move it. Thanks for pointing out.

**Pull request merged**